pkg/tfutils/tfWrapper.go

```diff
@@ -7,12 +7,14 @@ import (
 	"log"
 	"os"
 	"os/exec"
+	"strconv"
+	"strings"
 
 	"github.com/SAP/terraform-exporter-btp/pkg/toggles"
 	"github.com/spf13/viper"
 )
 
-func getIaCTool() (tool string, err error) {
+var getIaCTool = func() (tool string, err error) {
```

Comment:
I think transforming the function to a variable is not a good choice. This variable is global an could be changed by incident anywhere from the code. I think leaving the function as is and calling it is safer. I do not see why some sort of caching would be needed here.